air/src/options.rs

```diff
@@ -1,11 +1,28 @@
-use core::ops::Deref;
+use core::{marker::PhantomData, ops::Deref};
+use vm_core::crypto::hash::{Blake3_192, Blake3_256, Hasher};
 use winter_air::{FieldExtension, ProofOptions as WinterProofOptions};
 
-/// TODO: add docs
-#[derive(Clone)]
-pub struct ProofOptions(WinterProofOptions);
+// PROOF OPTIONS
+// ================================================================================================
 
-impl ProofOptions {
+/// A set of arguments to specify how the Miden proofs should be constructed.
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct ProofOptions<H>
+where
+    H: Hasher,
+{
+    hasher: PhantomData<H>,
+    options: WinterProofOptions,
+}
```

Comment:
I would do it differently:

I would introduce an enum for hash functions (e.g., `HashFunction`) we want to support. We probably don't need too many options. For now, we can have just 3: `Blake3_192`, `Blake3_256`, and `RPO256`.

Then, this struct could look something like:
```Rust
pub struct ProofOptions {
    options: WinterProofOptions,
    hash_fn: HashFunction,
}
```

Reply:
Done!